Mathlib/Data/Nat/Cast/Commute.lean

```diff
@@ -42,4 +42,14 @@ theorem _root_.Commute.ofNat_right [NonAssocSemiring α] (x : α) (n : ℕ) [n.A
     Commute x (OfNat.ofNat n) :=
   n.commute_cast x
 
+theorem commute_add_cast [NonAssocSemiring α] (r s : α) (n : ℕ) (h: Commute r s) :
```

Comment:
```suggestion
theorem commute_add_cast [NonAssocSemiring α] (r s : α) (n : ℕ) (h : Commute r s) :
```

Reply:
Can we make `NonAssocSemiring` a variable for this whole section actually?

Reply:
I added the variable declaration for the section and removed it from the theorems.